lib/bencher_adapter/src/results/adapter_results.rs

```diff
@@ -48,6 +55,31 @@ impl AdapterResults {
                             THROUGHPUT_RESOURCE_ID.clone() => json_metric
                         }
                     },
+                    AdapterMetricKind::Instructions(json_metric) => {
```

Comment:
Sure, that sounds good. It seems like we'll have to change AdapterResults as well, since it can only represent a single benchmark_name -> result mapping. Does that sound good?